include/windows/config.h

```diff
@@ -191,7 +191,8 @@
 
 /* Version number of package */
 #define _FI_EXP(s) #s
-#define VERSION _FI_EXP(FI_MAJOR_VERSION) "." _FI_EXP(FI_MINOR_VERSION) ".0"
+#define _FI_TO_STRING(s) _FI_EXP(s)
+#define VERSION _FI_TO_STRING(FI_MAJOR_VERSION) "." _FI_TO_STRING(FI_MINOR_VERSION) ".1a1"
```

Comment:
Why the double defines?

Also, I think this needs to be called out in the release process.  I've never updated this file as part of a new release.  I'll make that adjustment.